doc/conf.py

```diff
@@ -150,7 +150,12 @@
 html_theme_options = {}
 repository = "GenericMappingTools/pygmt"
 repository_url = "https://github.com/GenericMappingTools/pygmt"
-commit_link = f'<a href="{repository_url}/commit/{ __commit__ }">{ __commit__[:8] }</a>'
+if __commit__:
+    commit_link = (
+        f'<a href="{repository_url}/commit/{ __commit__ }">{ __commit__[:8] }</a>'
+    )
+else:
+    commit_link = ""
```

Comment:
Since we have this if-else block, could we be smart and point to the tag on GitHub?

Reply:
We'll need to detect whether the commit string is a hash like `a1b2c4d4` or a version tag like `v0.X.Y`, but the idea is to have something like so (note, don't commit as it won't work):

```suggestion
    commit_link = f'<a href="{repository_url}/tree/{ __commit__ }">{ __commit__}</a>'
```

which would e.g. point to https://github.com/GenericMappingTools/pygmt/tree/v0.3.1

Reply:
I was thinking about it too. It's OK to point to the specific tag, but the text would be "Revision v0.3.1", not "Release v0.3.1".

Reply:
"Revision v0.3.1" doesn't sound too bad actually (though it's not perfect either).

Reply:
Screenshot:
![image](https://user-images.githubusercontent.com/3974108/113536453-2aca2c80-95a4-11eb-8d9c-24e6a21c7101.png)

It links to https://github.com/GenericMappingTools/pygmt/releases/tag/v0.3.1

Reply:
Does this need to be backported to the gh-pages branch?

Reply:
Perhaps do the backport together with #1151, as mentioned in https://github.com/GenericMappingTools/pygmt/pull/1150#issuecomment-811053780.

Reply:
Ok, sounds like a good plan.